microprofile-jwt/src/test/resources/pki/README.md

```diff
@@ -1,15 +1,15 @@
 # Pieces of public key infrastructure necessary for JWT tests
 
+There are three main key pairs:
 
-## How to recreate
-```bash
-ssh-keygen -t rsa -b 2048 -m PEM -f key.private.pem #with empty passphrase!
-openssl pkcs8 -topk8 -v2 des3 -nocrypt \
-    -in key.private.pem \
-    -out key.private.pkcs8.pem
-openssl rsa -in key.private.pkcs8.pem -pubout -outform PEM -out key.public.pem
-rm key.private.pem key.private.pem.pub #cleanup
-```
+* `key.private.pkcs8.pem` and `key.public.pem` - this pair is the main key pair use in most tests, 2048 bits long
```

Comment:
I am wondering if some kind of prefix - like `main` or better (IMO) `default` wouldn't be good anyway. WDYT?

Reply:
I was wondering about this too. I think that if it would have "default" in its name someone might assume that it is chosen purely based on this... I can add this prefix/suffix if you want.

Reply:
Well, I think that what you just stated makes sense, so from my side no need for adding it.

Reply:
Yes, you reasoning makes sense and you also mentioned the fact in README, so from my side there's no need to add it.